src/styles/blocks/card-style.scss

```diff
@@ -0,0 +1,137 @@
+
+
+* {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  font-family: $primary-font;
+  font-weight: 400;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.container {
+  display: flex;
+  gap: 50px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-bcg-color;
+  padding: $card-padding;
+
+    &__img {
+      display: flex;
+      justify-content: center;
+      margin-bottom: 40px;
+    }
+
+    &__pic {
+      width: $card-img-width;
+      height: $card-img-height;
+    }
+
+    &__name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $primary-text-color;
+      margin-bottom: 4px;
+    }
+
+    &__code {
+      font-size: 10px;
+      line-height: 14px;
+      color: $secondary-text-color;
+      margin-bottom: 16px;
+    }
+
+    &__price {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 24px;
```

Comment:
Consider using only one direction of margins (top or bottom) to avoid margin collapse